### DISCUSSION TITLE
Can't pull through producst

### DISCUSSION THREAD

**Discussion by avocadesign** [statamic-rad-pack/shopify#249] [Help]
### Description

I've setup a test shopify site and connected it to Statamic with the this add-on. I think I've configured things correctly as per the docs. 

I can sync vendors/collections but products fail. If I do it via the CLI I get this message

php artisan shopify:import:all
================================================================
=================== IMPORT SHOPIFY PRODUCTS ====================
================================================================

   Symfony\Component\HttpFoundation\File\Exception\FileNotFoundException 

  The file "" does not exist

  at vendor/symfony/http-foundation/File/File.php:36
     32▕      */
     33▕     public function __construct(string $path, bool $checkPath = true)
     34▕     {
     35▕         if ($checkPath && !is_file($path)) {
  ➜  36▕             throw new FileNotFoundException($path);
     37▕         }
     38▕ 
     39▕         parent::__construct($path);
     40▕     }

      +65 vendor frames 

  66  artisan:13
      Illuminate\Foundation\Application::handleCommand(Object(Symfony\Component\Console\Input\ArgvInput))

Any idea what I've done wrong or how to fix this?

### Environment

Environment
Application Name: Nelson Honey
Laravel Version: 11.26.0
PHP Version: 8.3.12
Composer Version: 2.7.7
Environment: local
Debug Mode: ENABLED
URL: nelsonhoney.test
Maintenance Mode: OFF
Timezone: Pacific/Auckland
Locale: en

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: reverb
Cache: file
Database: sqlite
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Livewire
Livewire: v3.5.9

Statamic
Addons: 11
Sites: 1
Stache Watcher: Enabled (auto)
Static Caching: Disabled
Version: 5.35.0 PRO

Statamic Addons
jonassiewertsen/statamic-livewire: 3.8.1
lucky-media/seopulse-statamic: 1.4
reachweb/statamic-livewire-filters: 2.0.0
rias/statamic-data-import: 1.5.0
statamic-rad-pack/shopify: 4.0.6
statamic/collaboration: 1.0.0
statamic/seo-pro: 6.2.0
studio1902/statamic-peak-browser-appearance: 3.5.0
studio1902/statamic-peak-commands: 8.9.0
studio1902/statamic-peak-seo: 8.16.0
studio1902/statamic-peak-tools: 6.3.2

**Comment by ryanmitchell** [comment]
I would need the full stack trace - can you supply?
